api/Clients.json

```diff
@@ -123,11 +123,13 @@
             },
             "firefox": [
               {
-                "version_added": "54",
-                "notes": "<code>Client</code> objects returned in most recent focus order."
+                "version_added": "54"
               },
               {
-                "version_added": "44"
+                "version_added": "44",
+                "version_removed": "54",
+                "partial_implementation": true,
+                "notes": "Before Firefox 54, <code>Client</code> objects were not returned in the most recent focus order as required by the specification."
```

Comment:
We shouldn't have "Before Firefox 54" here.

Reply:
We actually have a fair number of notes that prefix "Before [Browser] [Version]" where the version matches the `version_removed` of a partial implementation (ex. `HTMLOptionElement` and `XMLDocument`), and we don't have a guideline on this.  Maybe we should create one?